### ISSUE TITLE
license

### ISSUE THREAD

**Issue by nobodywasishere** [cirosantilli/vcdvcd#31]
The Perl license is technically both the Artistic License and GPLv1+. As this is not clearly shown (and old), I would recommend updating the license of the repo to GPLv2+ explicitly, so that people using the repo know the proper licensing.

**Comment by cirosantilli** [comment]
The weird licensing is because this package was originally forked from someone else who forked in from Perl. I've added more details to the LICENSE file.

I don't think there is any easy justification for me to be able to change their license, it just depends on how much you believe this to count as a full rewrite. I've also added MIT license for everything I did now.

Does Artistic 1.0 cause any specific problems to you?
